### ISSUE TITLE
Clarify functionality

### ISSUE THREAD

**Issue by honglio** [ryanwalters/bootstrap-drawer#1]
Is this project provide a off-canvas navbar menu like in this example?  http://mediaflydesign.com/plugins/bootstrap-drawer/

**Comment by ryanwalters** [comment]
Yes, the drawer opens from the left. I have added an example page:
http://ryanwalters.github.io/bootstrap-drawer/

**Comment by honglio** [comment]
Have you tested it on mobile browser? Bootstrap fixed header doesn't fixed on mobile browser. 
![snapshot](http://www.luster.io/images/header_detached.png)
